adobe-esign-connector/processes/connector/TransientDocuments.p.json

```diff
@@ -167,5 +167,247 @@
           "connect" : { "id" : "f12", "to" : "f3" }
         } ],
       "connect" : { "id" : "f11", "to" : "f3" }
+    }, {
+      "id" : "f14",
+      "type" : "CallSubStart",
+      "name" : "uploadDocuments(List<File)",
+      "config" : {
+        "callSignature" : "uploadDocuments",
+        "input" : {
+          "params" : [
+            { "name" : "files", "type" : "java.util.List<java.io.File>", "desc" : "files to upload" }
+          ],
+          "map" : {
+            "out.files" : "param.files"
+          }
+        },
+        "result" : {
+          "params" : [
+            { "name" : "ids", "type" : "java.util.List<String>", "desc" : "ids of uploaded files" },
+            { "name" : "error", "type" : "ch.ivyteam.ivy.bpm.error.BpmError", "desc" : "holds error if one occurs, empty otherwise" }
+          ],
+          "map" : {
+            "result.ids" : "in.#ids",
+            "result.error" : "in.#error"
+          }
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 96, "y" : 360 }
+      },
+      "connect" : { "id" : "f22", "to" : "f21" }
+    }, {
+      "id" : "f15",
+      "type" : "CallSubEnd",
+      "visual" : {
+        "at" : { "x" : 776, "y" : 360 }
+      }
+    }, {
+      "id" : "f16",
+      "type" : "RestClientCall",
+      "config" : {
+        "bodyForm" : {
+          "File-Name" : "",
+          "Mime-Type" : "",
+          "File" : "in.file"
+        },
+        "path" : "/transientDocuments",
+        "clientId" : "6a243983-5f6d-4278-89ec-aab09fea161c",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "method" : "POST",
+        "statusErrorCode" : "ivy:error:rest:client",
+        "responseMapping" : {
+          "out.id" : "result.transientDocumentId"
+        },
+        "bodyObjectType" : "api.rest.v6.client.TransientDocumentsBody",
+        "resultType" : "api.rest.v6.client.TransientDocumentResponse",
+        "bodyInputType" : "FORM",
+        "bodyMediaType" : "multipart/form-data"
+      },
+      "visual" : {
+        "at" : { "x" : 384, "y" : 360 }
+      },
+      "boundaries" : [ {
+          "id" : "f17",
+          "type" : "ErrorBoundaryEvent",
+          "config" : {
+            "output" : {
+              "map" : {
+                "out" : "in",
+                "out.error" : "#error"
+              }
+            }
+          },
+          "visual" : {
+            "at" : { "x" : 416, "y" : 402 }
+          },
+          "connect" : { "id" : "f20", "to" : "f15", "via" : [ { "x" : 776, "y" : 400 } ] }
+        } ],
+      "connect" : { "id" : "f27", "to" : "f25" }
+    }, {
+      "id" : "f21",
+      "type" : "Script",
+      "name" : "pop file from list",
+      "config" : {
+        "output" : {
+          "code" : "in.file = in.files.removeGet(0);"
```

Comment:
please check null or empty before modify the collection to avoid outofbound exception
![image](https://github.com/axonivy-market/adobe-esign-connector/assets/83745591/6b0dbfed-1b79-4c9a-b6c2-f5954a92590a)

Reply:
Hi @nqhoan-axonivy , thanks for the feedback, should be addressed already 😄